src/pages/Home/hooks/useStoreList.ts

```diff
@@ -0,0 +1,26 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { StoreListParams } from '@/api/shop/entity';
+import { getStoreList } from '@/api/shop/index';
+
+interface UseStoreListParams extends StoreListParams {
+  category?: number | null;
```

Comment:
```suggestion
  category?: number;
```
null 허용 없애고
```
const [selectedCategory, setSelectedCategory] = useState<CategoryType>(1);
```
카테고리에서 null 타입 제외해두는 방식으로도 동일하게 동작하는 것 같은데 아닐까요?

Reply:
맞습니다! 카테고리는 필터와 다르게 무조건 값이 존재하기 때문에 null을 없애도 될 것 같습니다!